<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1041</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="opcja2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>opcja 2 - prędkość</string>
      </property>
     </widget>
     <widget class="QPushButton" name="opcja1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>opcja 1 - temperatura</string>
      </property>
     </widget>
     <widget class="QPushButton" name="opcja3">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>200</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>opcja 3 - podawanie medium</string>
      </property>
     </widget>
     <widget class="QPushButton" name="opcja4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>opcja 4 - zdjęcie</string>
      </property>
     </widget>
     <widget class="QPushButton" name="opcja5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>300</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>opcja 5 - zapisz </string>
      </property>
     </widget>
     <widget class="QPushButton" name="opcja6">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>300</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="option3_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK TO MENU</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>441</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Uroob</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>IMU 1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Initialized correct:</string>
       </property>
      </widget>
      <widget class="QLabel" name="imuinit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>31</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Uroob</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="imu1x">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>64</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>21</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Uroob</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="imu1y">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>50</y>
         <width>64</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>40</y>
         <width>31</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Uroob</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="imu1z">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>50</y>
         <width>64</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>540</y>
        <width>171</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>REFRESH DATA</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="option3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK TO MENU</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>POMPOWANIE MEDIUM </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CHIP 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CHIP 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>PWM SZYBKOSC POMPOWANIA MEDIUM:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CZAS OPERACJI [s]: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>190</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>PWM SZYBKOSC POMPOWANIA MEDIUM:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>270</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CZAS OPERACJI [s]: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="setbtn_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>290</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setbtn_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QSlider" name="foodspeed1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>240</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="foodtime1">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>270</y>
        <width>42</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="foodtime2">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>270</y>
        <width>42</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="foodspeed2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>240</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>GO BACK TO MENU</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string> USTAW PREDKOSC OBROTOWA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>160</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CHIP 2 [obr/min]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CHIP 1 [obr/min]</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>220</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="setbtn_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setbtn_7">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>250</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="option2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK TO MENU</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>USTAWIENIA TEMPERATURY</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CHIP 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CHIP 2</string>
      </property>
     </widget>
     <widget class="QSlider" name="slider1">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="slider2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd1">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>190</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>270</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="setbtn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="option1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK TO MENU</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FOTOGRAFIA BIOCHIPA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CHIP 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>170</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CHIP 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>KROTNOSC ZDJEC [zdj/24h]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>210</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Uroob</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>KROTNOSC ZDJEC [zdj/24h]</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxphoto1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>42</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxphoto2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>250</y>
        <width>42</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="setbtn_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setbtn_3">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>250</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Aktualna wartość: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>290</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Aktualna wartość:</string>
      </property>
     </widget>
     <widget class="QLabel" name="photolabel1">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="photolabel2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>290</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1316</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
